### PR TITLE
[WIP] docs(cookbook-a1-a2): quick reference

### PR DESCRIPTION
**Pull request by wardbell**
WIP DO NOT MERGE

### CODE REVIEW

File: public/docs/ts/latest/cookbook/a1-a2-quick-reference.jade [hunk 1]
```diff
@@ -0,0 +1,773 @@
...
+      code-example.
+         &lt;h2>{{ vm.title }}&lt;/h2>
+      :marked
+        In Angular&nbsp;1, an expression in curly braces denotes one-way binding.
+        This binds the value of the element to a property in the controller
+        associated with this template
+        
+        When using the `controller as` syntax,
+        the binding is prefixed with the controller alias (vm) because we
+        have to be specific about the source of the binding.
```
Comment (DeborahK) [inline public/docs/ts/latest/cookbook/a1-a2-quick-reference.jade]: Should each of these A2 columns have links to more information in the developer guide section of the documentation?

File: public/docs/ts/latest/cookbook/a1-a2-quick-reference.jade [hunk 2]
```diff
@@ -0,0 +1,773 @@
...
+    td
+      :marked
+        ### Bindings/Interpolation
+      code-example.
+         &lt;h2>{{ title }}&lt;/h2>
+      :marked
+        In Angular&nbsp;2, a template expression in curly braces still denotes one-way binding.
+        This binds the value of the element to a property of the component.
+        The context of the binding is implied and is always the
+        associated component, so it needs no reference variable.
```
Comment (DeborahK) [inline public/docs/ts/latest/cookbook/a1-a2-quick-reference.jade]: Should each of these A2 columns have links to more information in the developer guide section of the documentation?

File: public/docs/ts/latest/cookbook/a1-a2-quick-reference.jade [hunk 3]
```diff
@@ -0,0 +1,773 @@
...
+        associated with this template
+        
+        When using the `controller as` syntax,
+        the binding is prefixed with the controller alias (vm) because we
+        have to be specific about the source of the binding.
+    td
+      :marked
+        ### Bindings/Interpolation
+      code-example.
+         &lt;h2>{{ title }}&lt;/h2>
```
Comment (DeborahK) [inline public/docs/ts/latest/cookbook/a1-a2-quick-reference.jade]: NOTE: The A2 code will be changed to actual code examples once I have confirmation that I'm on the right track with this.

**Comment by wardbell** [comment]
Closed as part of cleanup. New PR is #786

**Pull request closed without merging**